geth/common/types.go

```diff
@@ -395,6 +395,12 @@ type TestConfig struct {
 	}
 }
 
+// NotifyResult is a JSON returned from notify message
+type NotifyResult struct {
+	Status bool   `json:"status"`
+	Error  string `json:"error"`
```

Comment:
Add `omitempty`? I guess there is no point it passing this field with an empty string. 

It's still something that we need to sit and talk about with ClojureTeam as it's pretty inconsistent.

Reply:
You can start: https://docs.google.com/document/d/1BBit9U2_-_B12NczUiOGzrGMU8z1vQeAAuqE2HVIYKc :)

Reply:
What do you think about making this doc more general and actually specify input and output formats? Specifying error codes won't change much as we still won't have a complete picture what kind of data we exchange.

Reply:
I've just started it when we had a similar question so that somebody filled it :) I'll make an issue for that shortly.

Reply:
I see :) Ok, let's make an issue to track progress, keep all related issues in one place and define acceptance criteria. It will be super useful when we decide to switch to gomobile.

Reply:
#408

Reply:
'omitempty' for error added.